confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -94,9 +152,15 @@ def _send_request(self, url, method='GET', body=None, headers={}):
         _headers.update(headers)
 
         response = self._session.request(method, url, headers=_headers, json=body)
+
+        # Returned by Jetty not SR so the payload is not json encoded
```

Comment:
Might be better to wrap the json parsing in a try-block for all 4xx

Reply:
Any comment on this? Still seems like a valid concern

Reply:
Sorry missed this one. I like this idea I did something similar in the CCloud CLI as well.